src/app/components/YourLocationInput/YourLocationInput.tsx

```diff
@@ -12,37 +12,90 @@ export default function YourLocationInput({
 }: YourLocationInputProps): JSX.Element {
   return (
     <Container>
-      <span>Your locations are set to:</span>
-      <br />
+      <Text>Your locations are set to:</Text>
+      <svg
```

Comment:
Why didn't you make a component out of this icon?

Reply:
bc it was too late :) but you are rigth, i created a component for it :)